### ISSUE TITLE
Toolbar item 

### ISSUE THREAD

**Issue by GoogleCodeExporter** [heron-mc/heron-mc#409]
```
Maybe there´s an extJS way to launch a (possibly hidden) toolbar item like the 
following: 

{type:"any",options: {
text:'Measures',
tooltip: 'Measure of lengthe and area',
iconCls: "bmenu",
menu:{
    items:[
                  {type: "measurelength"},
              {type: "measurearea"}
]
}
}
}


but it doesn't work for me with already defined functions (like "measurelength" 
or "measurearea") only with new functions 
(http://lib.heron-mc.org/heron/1.0.5/examples/toolbar-item/index.html)...

The feature would be very nice to clean the toolbar.

R
```

Original issue reported on code.google.com by `roberto....@gmail.com` on 30 Dec 2014 at 9:02

**Comment by justb4** [comment]
I think I now understand and may be quite useful to bundle toolbar items under a menu. Possibly we should add a toolbar item type: 'composite' that allows subitems to be configured just like topbar items. Think that is what your config suggests.

**Comment by rmarzocchi84** [comment]
Yes. I think this can be really useful for toolbar with a loto of options





Il 06/03/2016 13:51, Just van den Broecke ha scritto:
>
> I think I now understand and may be quite useful to bundle toolbar 
> items under a menu. Possibly we should add a toolbar item type: 
> 'composite' that allows subitems to be configured just like topbar 
> items. Think that is what your config suggests.
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/heron-mc/heron-mc/issues/409#issuecomment-192884810>.
>

-- 
Eng. Roberto Marzocchi, PhD
GIS Project Coordinator
Gter srl Innovazione in Geomatica, Gnss e Gis (Unige spin-off)
Piazza De Marini 3/61 - 16123 Genova
P.IVA/CF 01998770992
ph: 010-8694830 - mob: 349-8786575
E-mail: roberto.marzocchi@gter.it
skype: roberto.marzocchi84
www.gter.it

--
Gter social
www.twitter.com/Gteronline - www.facebook.com/Gteronline - https://plus.google.com/+GterIt/posts
www.linkedin.com/company/gter-srl-innovazione-in-geomatica-gnss-e-gis

-----------------------------------------------------------------
Please consider the environment before printing this email!
